### ISSUE TITLE
RNAlocmin compile error

### ISSUE THREAD

**Issue by mmiladi** [ViennaRNA/ViennaRNA#13]
llvm compiler for bio-conda travis CI complains about RNAlocmin: 
```
main.cpp:366:18: error: variable length array of non-POD element type 'nodeT'
nodeT nodes[num];
```
https://travis-ci.org/bioconda/bioconda-recipes/jobs/274279776#L4184

Which seems to refer to: 
https://github.com/ViennaRNA/ViennaRNA/blob/21610fce6202b77da3ab1093c2dfa1c3943f5cfa/src/RNAlocmin/main.cpp#L366

Dynamic allocation of nodeT array may solve the problem.

**Comment by s-will** [comment]
Thanks @mmiladi. I suggested a fix to Ronny.

**Comment by RaumZeit** [comment]
Fixed in v2.4.2
